web-server/src/components/Service/SystemLog/PlainLog.tsx

```diff
@@ -1,9 +1,40 @@
 import { Line } from '@/components/Text';
 
-export const PlainLog = ({ log }: { log: string; index: number }) => {
+interface PlainLogProps {
+  log: string;
+  index: number;
+  searchQuery?: string;
+}
+
+const HighlightedText = ({ text, searchQuery }: { text: string; searchQuery?: string }) => {
```

Comment:
This component is duplicated. Figure out a way to avoid duplication to avoid future code drift.